test/apps/cloudAppClient/manager-eventRequest.test.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+var test = require('tape')
+var helper = require('../../helper')
+var EventEmitter = require('events')
+var Manager = require(`${helper.paths.apps}/cloudappclient/manager.js`)
+var Skill = require(`${helper.paths.apps}/cloudappclient/skill`)
+var eventRequestMap = require(`${helper.paths.apps}/cloudappclient/eventRequestMap.json`)
+
+class EventRequest extends EventEmitter {
+  ttsEvent (name, appId, itemId, cb) {
+    var res = '{}'
+    cb(res)
```

Comment:
No need to declare the `res`: `cb('{}')`.

Reply:
StandardJS does not allow this format: `cb('{}')`
we need to add rule `no-callback-literal: 0` in .eslintrc.js file.
> [standard] Unexpected literal in error position of callback. (standard/no-callback-literal) [standard/no-callback-literal]

Reply:
That's because the `cb` tells standardjs this function is a callback, and at Node.js, a callback's first argument must be an error.